ophyd/pv_positioner.py

```diff
@@ -296,3 +301,151 @@ def done_moving(**kwargs):
         else:
             self.setpoint.put(position, wait=False,
                               callback=done_moving)
+
+
+class PVPositionerComparator(PVPositioner):
+    """
+    PV Positioner with a software done signal.
+
+    The done state is set by a comparison function defined in the class body.
+    The comparison function takes two arguments, readback and setpoint,
+    returning True if we are considered done or False if we are not.
+
+    This class is intended to support `PVPositionerIsClose`, but exists to
+    allow some flexibility if we want to use other metrics for deciding if
+    the PVPositioner is done.
+
+    Internally, this will subscribe to both the `setpoint` and `readback`
```

Comment:
```suggestion
    Internally, this will subscribe to both the ``setpoint`` and ``readback``
```

(Single backtick means an `:any:` reference, right? If I'm misremembering, ignore this)

Reply:
Looks like default_role is not set for ophyd: https://github.com/bluesky/ophyd/blob/cf8e8750e07689fe8335bdf477ad914cd5c31d3e/docs/source/conf.py#L81-L82